src/contact.ts

```diff
@@ -150,25 +167,27 @@ export class Contact implements Sayable {
       /**
        * @see 1. https://github.com/Chatie/webwx-app-tracker/blob/7c59d35c6ea0cff38426a4c5c912a086c4c512b2/formatted/webwxApp.js#L3243
        * @see 2. https://github.com/Urinx/WeixinBot/blob/master/README.md
+       * @ignore
        */
       // tslint:disable-next-line
       official:      !!rawObj.UserName && !rawObj.UserName.startsWith('@@') && !!(rawObj.VerifyFlag & 8),
       /**
        * @see 1. https://github.com/Chatie/webwx-app-tracker/blob/7c59d35c6ea0cff38426a4c5c912a086c4c512b2/formatted/webwxApp.js#L3246
+       * @ignore
        */
       special:       specialContactList.indexOf(rawObj.UserName) > -1 || /@qqim$/.test(rawObj.UserName),
     }
   }
 
   /**
-   * Get the weixin number from a contact
+   * Get the weixin number from a contact.
+   *
    * Sometimes cannot get weixin number due to weixin security mechanism, not recommend.
-   * @returns {string | null}
    *
+   * @deprecated
+   * @returns {string | null}
    * @example
-   * ```ts
```

Comment:
Could please confirm it for me that we do not need "\`\`\`ts" block inside `@example`?